src/User32.Desktop/User32.cs

```diff
@@ -68,6 +69,16 @@ public static partial class User32
         [DllImport(nameof(User32), SetLastError = true)]
         public static extern int GetWindowLong(IntPtr hWnd, WindowLongIndexFlags nIndex);
 
+        [DllImport(nameof(User32), SetLastError = true, CharSet = CharSet.Auto)]
+        public static extern int GetClassName(IntPtr hWnd, StringBuilder lpClassName, int nMaxCount);
+
+        [DllImport(nameof(User32), SetLastError = true)]
+        public static extern uint GetWindowThreadProcessId(IntPtr hWnd, out uint lpdwProcessId);
```

Comment:
Yes, there is that balance between weird and easy to use. As I've never seen a PID that is in excess of `int.MaxValue`, I suspect that's why .NET also just decided that `int` was good enough.